typescript/sdk/src/consts/chainMetadata.ts

```diff
@@ -46,6 +46,34 @@ export const alfajores: ChainMetadata = {
   rpcUrls: [{ http: 'https://alfajores-forno.celo-testnet.org' }],
 };
 
+export const ancient8testnet: ChainMetadata = {
+  blockExplorers: [
+    {
+      apiUrl: 'https://testnet.a8scan.io/api',
+      family: ExplorerFamily.Blockscout,
+      name: 'a8scan',
+      url: 'https://testnet.a8scan.io',
+    },
+  ],
+  chainId: 2863311531,
+  blocks: {
+    confirmations: 1,
+    estimateBlockTime: 2,
+    reorgPeriod: 0,
+  },
+  displayName: 'Ancient 8 Testnet',
```

Comment:
Please add a `displayNameShort` for the explorer
Bonus points for also adding an svg icon but it's nbd, I'll get to it later